nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/CheckedInDocumentListener.java

```diff
@@ -55,7 +56,8 @@ public void handleEvent(Event event) {
                 return;
             }
             DocumentRef versionRef = (DocumentRef) ctx.getProperty("checkedInVersionRef");
-            if (versionRef instanceof IdRef) {
+            if (versionRef instanceof IdRef && Framework.getService(ConfigurationService.class).isBooleanPropertyFalse(
```

Comment:
Do
`boolean facetedTags = Framework.getService(ConfigurationService.class).isBooleanPropertyTrue(TagServiceImpl.FACETED_TAG_SERVICE_ENABLED)`
then
`if (versionRef instanceof IdRef && !facetedTags)`
for clarity.